core/src/main/java/org/apache/jsieve/comparators/AsciiCasemap.java

```diff
@@ -40,6 +40,8 @@ public AsciiCasemap() {
      * @see org.apache.jsieve.comparators.Equals#equals(String, String)
      */
     public boolean equals(String string1, String string2) {
+	if (string1 == null && string2 == null) return true;
```

Comment:
Do you think it would be possible to you to propose tests for this changeset?

Maybe we can write a AsciiCasemapTest class, and test :
 - null, null
 - "a", null
 - null, "a"
 - "a", "b"
 - "a", "a"
 - "a", "A"